SECURITY.md

```diff
@@ -1,5 +1,16 @@
 # Security Policy
 
+## Reporting a Vulnerability
+
+We are at the very beginning of our journey (as of March 2025). While we build on the shoulders of giants like Rust and
+Polkadot SDK, following industry standards and best practices, it would be overly optimistic to expect a completely
+secure system at this stage. Our goal is to communicate transparently about any issues and address them swiftly and
+carefully.
+
+If you discover a vulnerability or have a security concern, please report it by opening [open a new issue](https://github.com/QuantumFusion-network/qf-solochain/issues/new)
+in the GitHub repository. A detailed explanation, including reproduction steps or a reference to the vulnerability source
+code, would be greatly appreciated!
```

Comment:
It's common practice not to disclose new vulnerabilities publicly. I recommend providing an email for vulnerability reports instead.

Reference:
https://github.com/rust-lang/rust/security

For comparison, see our page:
https://github.com/QuantumFusion-network/qf-solochain/security

Reply:
Agree. I used polkadot-sdk as a reference and they have 3 emails: for reporting a case when a maintainer violates the code of conduct, for vulnerability reports and for bug bounty requests. For our case public vulnerability reporting seems fine until mainnet, I mean there is nothing it could hurt. At the same time we don't have email server for that purpose setup yet, but want to welcome contributions as early as possible. What do you think?